awkward-cpp/pyproject.toml

```diff
@@ -91,22 +91,22 @@ pytest -p no:cacheprovider \
 """]
 skip = [
     "pp*win*",
-    "pp*-manylinux_i686",
 ]
 test-skip = [
-    "*universal2:arm64",
```

Comment:
We should be building a pair of native wheels, not universal2. Just old cleanup.